tests/migration_test.go

```diff
@@ -1446,11 +1447,22 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 					Skip("Skip DataVolume tests when CDI is not present")
 				}
 			})
+
 			It("[test_id:3239]should reject a migration of a vmi with a non-shared data volume", func() {
-				dataVolume := libstorage.NewDataVolumeWithRegistryImport(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskAlpine), util.NamespaceTestDefault, k8sv1.ReadWriteOnce)
+				sc, exists := libstorage.GetRWOFileSystemStorageClass()
+				if !exists {
+					Skip("Skip test when Filesystem storage is not present")
+				}
+
+				dataVolume := libdv.NewDataVolume(
+					libdv.WithNamespace(util.NamespaceTestDefault), // for deletion
```

Comment:
it can be deleted right now, and then the actual datavolume used in line 1465:
`				dataVolume, err := virtClient.CdiClient().CdiV1beta1().DataVolumes(util.NamespaceTestDefault).Create(context.Background(), dataVolume, metav1.CreateOptions{})`

But I do not insist on doing it in current PR

Reply:
and I think this pattern could be repeated in other tests